web/src/store/index.ts

```diff
@@ -272,7 +296,26 @@ const {
     },
     updateSelectedPins(context, pins: Pin[]) {
       const { commit } = rootActionContext(context);
-      commit.setSelectedPins(pins);
+      const circlePinDatasetTypes = [
+        'tile_overlay',
+        'non_tiled_image',
+      ];
+      const isVisualization: Array<boolean | Dataset> = pins.map((pin: Pin) => {
+        const childDataset: Dataset | undefined = store.state.currentDatasets.find(
+          (dataset: Dataset) => dataset.id === pin.child,
+        );
+        if (childDataset
+          && childDataset?.dataset_type
+          && !circlePinDatasetTypes.includes(childDataset.dataset_type)) {
+          return childDataset;
+        }
+        return false;
+      });
+      const circlePins = pins.filter((pin: Pin, index: number) => !isVisualization[index]);
+      const visualizations = isVisualization.filter((vis) => vis !== false);
+
+      commit.setSelectedPins(circlePins);
+      commit.setSelectedVisualizations(visualizations);
```

Comment:
There's certainly a lot happening here now that I'm not sure needs to. I'm wondering if these special visualizations really belong to Pins at all. It doesn't seem like the fit the cork board metaphor of pinning something to the larger dataset, since (at least in the nucleus/glands example) the visualizations apply to the whole of the base dataset.

I think having these just be datasets that get handled specially makes more sense than them being Pins. Let me know if you have any thoughts on this. I think it'd be fine to get this in as a first go but I think there are design considerations that would be worth discussing.

Reply:
I agree, and I took a second pass at this design in #201, though the backend of that design remained unchanged. I do think we should return to this topic.

Reply:
Looks like #201 was closed in favor of #204. I'm fine with using pins for these two PRs, but I've opened #206 to try and capture some of my feelings with the current design. At this point I'd like to get this feature merged.